### PR TITLE
WIP: [#6233] Pm 2022 07 profile with dummy data

### PR DESCRIPTION
**Pull request by phillimorland**
this also has design cleaning in as it was just a bit annoying

### CODE REVIEW

File: components/ButtonAvatar.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { TouchableOpacity, Image } from 'react-native'
+import { styles } from './ButtonAvatar.styles'
+
+export const AvatarCircle = (props) => {
```
Comment (goapunk) [inline components/ButtonAvatar.js:5]: do we use that somewhere else than in this file ? If not we don't need to export it, right?

File: containers/Ideas/ExplorePage.js [hunk 2]
```diff
@@ -29,9 +29,17 @@ export const ExplorePage = (props) => {
...
+    props.navigation.navigate('ProfileScreen', {name: 'Bob'})
+  }
+
   return (
     <View style={styles.container}>
-      <ButtonSignOut></ButtonSignOut>
+      <ButtonAvatar
+        labelText="profile"
+        hintText="click to go to profile and settings"
+        onPress={() => toProfile()}
```
Comment (goapunk) [inline containers/Ideas/ExplorePage.js:41]: would  just `{toProfile}` do or do we need the arrow funtion here?

File: containers/User/ProfileScreen.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { View } from 'react-native'
+import { styles } from './ProfileScreen.styles'
```
Comment (fuzzylogic2000) [inline containers/User/ProfileScreen.js:3]: Did you forget to commit that file?

Comment (phillimorland) [reply containers/User/ProfileScreen.js:3]: dammit, yes, will push in a minute

Comment (phillimorland) [reply containers/User/ProfileScreen.js:3]: @fuzzylogic2000 sorry should be good now, it's a re-write though so let me know if forgot anything

**Comment by fuzzylogic2000** [comment]
I started adding the API stuff to this PR and made it WIP with that. Hope, that's fine!

**Comment by phillimorland** [comment]
@fuzzylogic2000 cool, thank you! i'll go on with @goapunk review comments and update tests

**Comment by fuzzylogic2000** [comment]
I made a bit of a mess with @phillimorland s PR now. And added the commit that she had to do in another branch back to this one now. 
@khamui @Rineee Could you already look at this and say what needs fixing before we merge? I think a merge would be easier to then go off into the different directions. Buuuut actually, no, the user image needs fixing for sure, when the only hard-coded user doesn't have an image this will break.

**Comment by fuzzylogic2000** [comment]
Ok, I will close this in favour of #371 now and make a new branch for the API info stuff.

**Pull request closed without merging**